SSR-3374 Add icon.png and css on format setting plugin guide article (#1339)
kintone-labs/kintone-ui-component: docs/document/docs/guides/format-setting-plugin.md

├── manifest.json
```

> ## Create a Plug-in Icon
> - The icon image for the Plug-in.
> - This file is required.
> - Use the following file types: png, jpg, gif, bmp
> - The maximum file size is 20MB.
> - Please add icon file into folder ``image/icon.png``

Reviewer: I think you don't need to write precise requirements for icon.
You can link the detail using the official guide.
Here is my suggestion.

```suggestion
## Create a plug-in icon
Create a plug-in icon following the [guide](https://kintone.dev/en/plugins/introduction-to-plug-ins/steps-for-plug-in-development/#creating-each-file).<br>
Place the icon file (icon.png) in the image folder.
```

> ## Create a config.css file
> - The stylesheet initiated on the Plug-in.
> - You can follow the Kintone stylesheet if you want to use the Kintone design.
> - The maximum file size is 20MB.
> - This is optional, and can be ignored. But following the example below, You can create a file ``css/config.css``

Reviewer: I think it's better to unify the format with config.js.
Please place the description under config.js explanations.

```suggestion
Place the CSS file (config.css) described below in the css folder.<br>

***config.css***

```

```css
.kuc-dropdown__group > .kuc-dropdown__group__toggle {
  width: 200px;

> }

Reviewer: I think this won't work now because the class name will include the version number now.
I think we can remove this setting.

#kuc_cancel_button {
  margin-right: 16px;
}

#kuc_dialog_footer {
  text-align: center;
}

#kuc_dialog_cancel_button {
  margin-right: 16px;
}
```

> 

Reviewer: I wanna suggest ordering like this as the actual steps to implement.

- Create a manifest file
- Create a plug-in icon
- HTML customization
- JavaScript and CSS customization (Settings page)
  - config.js
  - config.css
- JavaScript and CSS customization (Kintone app page)
  - desktop.js

Kintone UI Component will be used for this customization on the plug-in settings page, so describe the path of the UMD file in the config object.<br>
See [Quick Start](../getting-started/quick-start.md) for how to upload a UMD file.

> ## Create a Plug-in Icon
> - Create a plug-in icon following the [guide](https://kintone.dev/en/plugins/introduction-to-plug-ins/steps-for-plug-in-development/#creating-each-file).<br>
> - Place the icon file (icon.png) in the image folder.

Reviewer: Please change section title to lower case.
Please write the contents without bullet points.

```suggestion
## Create a plug-in icon
Create a plug-in icon following the [guide](https://kintone.dev/en/plugins/introduction-to-plug-ins/steps-for-plug-in-development/#creating-each-file).<br>
Place the icon file (icon.png) in the image folder.
```

  // Write the process here
})(kintone.$PLUGIN_ID);
```

Place the CSS file (config.css) described below in the css folder.<br>

***config.css***

```css
#kuc_cancel_button {
  margin-right: 16px;
}

#kuc_dialog_footer {
  text-align: center;
}

#kuc_dialog_cancel_button {
  margin-right: 16px;
}

> ```

Reviewer: I think it's better to place css description under all the js customization explanations following the actual customization steps.
How do you think?
